kafka-rest/src/main/java/io/confluent/kafkarest/controllers/KafkaFutures.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.controllers;
+
+import java.util.concurrent.CompletableFuture;
+import org.apache.kafka.common.KafkaFuture;
+
+final class KafkaFutures {
+
+  private KafkaFutures() {
+  }
+
+  static <T> CompletableFuture<T> toCompletableFuture(KafkaFuture<T> kafkaFuture) {
```

Comment:
This looks useful. Did you consider contributing to Apache Kafka, so KafkaFuture object will support this conversion out of the box?

Reply:
I've never done an AK PR. Sounds exciting :)